.github/workflows/ci.yml

```diff
@@ -1,45 +1,24 @@
 name: CI
 
-# TODO:
-# - Run unit tests for all platforms (Blocked by XCode build file creation.)
-# - Reduce duplicate code (Blocking: https://github.community/t5/GitHub-Actions/Support-for-YAML-anchors/td-p/30336)
-# - Investigate using scoop as package manager on windows
-# - Cache build files on all systems (Looking at you OSX)
-# - Cache development libraries on windows
-# - Run windows builds as matrix with different mingw versions
-# - Construct even more intelligent workflows
-#    e.g.
-#      data/* -> use "latest" build for triggering branch, only run parse and/or acceptance tests
-#      source/* -> build new binary, run all tests, cache binary
-#      PR opened -> upload artifact ZIPs to PR as comment, and edit/replace as new commits come in
-# - Windows: Allow programmers to use their own copy of development libs without hacking a temporary URL in the windows step
-
 on:
   push:
-    # Run for pushes to master or a release branch, e.g. a PR was merged ...
```

Comment:
Why remove these? same for the other lines

Reply:
oops collateral damage